src/main/java/org/mqttbee/mqtt/handler/MqttChannelInitializer.java

```diff
@@ -54,9 +57,21 @@
 
     @Override
     protected void initChannel(final SocketChannel channel) {
-        final ChannelComponent channelComponent = ChannelComponent.create(channel, clientData);
+        init(channel);
+        addMqttHandlers();
+    }
+
+    public ChannelPipeline getPipeline() {
```

Comment:
Not needed, as MqttWebSocketsChannelInitializer can access channel.pipeline()

Reply:
done